### PR TITLE
discover a detected test

### PR DESCRIPTION
**Pull request by Carol7102**
The flaky test org.jitsi.jicofo.auth.ShibbolethAuthenticationAuthorityTest.testShibbolethAuthenticationModule was deleted by developer at commit https://github.com/jitsi/jicofo/commit/adaedbfd013e14462f53f7ffe616a3a186a26d51.
The flaky test org.jitsi.jicofo.auth.XMPPAuthenticationAuthorityTest.testXmppDomainAuthentication was renamed and moved to kotlin at commit https://github.com/jitsi/jicofo/commit/305b5125537f75db0c8f26bae80e42871a943ac6.
The flaky test org.jitsi.jicofo.RolesTest.testPassModeratorRole was deleted at commit https://github.com/jitsi/jicofo/commit/305b5125537f75db0c8f26bae80e42871a943ac6.
The flaky test org.jitsi.jicofo.XmppTest.testAllocateConference was renamed and moved to kotlin at commit https://github.com/jitsi/jicofo/commit/adaedbfd013e14462f53f7ffe616a3a186a26d51.

### CODE REVIEW

**Comment by darko-marinov** [comment]
Do not delete the existing link from the last column (`Notes`) but just add a new link separated with `;`. Can you send here (or via email/Campuswire) the direct link to where the test was deleted, not the entire commit but the exact line for this test?

**Comment by Carol7102** [comment]
> Do not delete the existing link from the last column (`Notes`) but just add a new link separated with `;`. Can you send here (or via email/Campuswire) the direct link to where the test was deleted, not the entire commit but the exact line for this test?

Sorry that I found wrong commit before, I have updated the right commit where the test was deleted, and the changed file is src/test/java/org/jitsi/jicofo/auth/ShibbolethAuthenticationAuthorityTest.java, the line for this test is line 64 - line 176.

**Comment by darko-marinov** [comment]
> the changed file is src/test/java/org/jitsi/jicofo/auth/ShibbolethAuthenticationAuthorityTest.java, the line for this test is line 64 - line 176.

What's the link to that?

**Comment by Carol7102** [comment]
> > the changed file is src/test/java/org/jitsi/jicofo/auth/ShibbolethAuthenticationAuthorityTest.java, the line for this test is line 64 - line 176.
> 
> What's the link to that?
Hi, this is the link and the changed file is src/test/java/org/jitsi/jicofo/auth/ShibbolethAuthenticationAuthorityTest.java, the line for this test is line 64 - line 176:
https://github.com/jitsi/jicofo/commit/adaedbfd013e14462f53f7ffe616a3a186a26d51

**Comment by darko-marinov** [comment]
Sorry I didn't ask for the link to the commit but directly to the change. For example, in the commit you linked, this change https://github.com/jitsi/jicofo/commit/adaedbfd013e14462f53f7ffe616a3a186a26d51#diff-96feffacb07976b8b019f1657b787c174cb88d7a8a0e7084098a4de39f87d0e2 is for the file `TestConference.java`. I want a link to the change where you claim the test was `Deleted`. (My question is inspired partly by not being able to find the class name in the previous commit you reported.)

**Comment by Carol7102** [comment]
> Sorry I didn't ask for the link to the commit but directly to the change. For example, in the commit you linked, this change [jitsi/jicofo@adaedbf#diff-96feffacb07976b8b019f1657b787c174cb88d7a8a0e7084098a4de39f87d0e2](https://github.com/jitsi/jicofo/commit/adaedbfd013e14462f53f7ffe616a3a186a26d51#diff-96feffacb07976b8b019f1657b787c174cb88d7a8a0e7084098a4de39f87d0e2) is for the file `TestConference.java`. I want a link to the change where you claim the test was `Deleted`. (My question is inspired partly by not being able to find the class name in the previous commit you reported.)

Thanks for pointing it out, the link to the test deleted is https://github.com/jitsi/jicofo/commit/adaedbfd013e14462f53f7ffe616a3a186a26d51#diff-cbdcfe3c568b2df0df5a98eb4d9a44d62d823de36e01d724efe8315e28897aac.

**Comment by Carol7102** [comment]
> Sorry I didn't ask for the link to the commit but directly to the change. For example, in the commit you linked, this change [jitsi/jicofo@adaedbf#diff-96feffacb07976b8b019f1657b787c174cb88d7a8a0e7084098a4de39f87d0e2](https://github.com/jitsi/jicofo/commit/adaedbfd013e14462f53f7ffe616a3a186a26d51#diff-96feffacb07976b8b019f1657b787c174cb88d7a8a0e7084098a4de39f87d0e2) is for the file `TestConference.java`. I want a link to the change where you claim the test was `Deleted`. (My question is inspired partly by not being able to find the class name in the previous commit you reported.)

I have updated three other tests that were either deleted or moved. Appreciate your assistance!

File: pr-data.csv [hunk 1]
```diff
@@ -3943,10 +3943,10 @@ https://github.com/jenkinsci/remoting,abf0455a68ad6c52a57e912bb89d51f883f77542,.
...
-https://github.com/jitsi/jicofo,2cae36e1e6781b438e52156d82cc32f8dea021b9,.,org.jitsi.jicofo.RolesTest.testPassModeratorRole,OD,,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/83
-https://github.com/jitsi/jicofo,2cae36e1e6781b438e52156d82cc32f8dea021b9,.,org.jitsi.jicofo.XmppTest.testAllocateConference,OD,,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/83
+https://github.com/jitsi/jicofo,2cae36e1e6781b438e52156d82cc32f8dea021b9,.,org.jitsi.jicofo.auth.ShibbolethAuthenticationAuthorityTest.testShibbolethAuthenticationModule,OD,Deleted,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/83;https://github.com/jitsi/jicofo/commit/adaedbfd013e14462f53f7ffe616a3a186a26d51
+https://github.com/jitsi/jicofo,2cae36e1e6781b438e52156d82cc32f8dea021b9,.,org.jitsi.jicofo.auth.XMPPAuthenticationAuthorityTest.testXmppDomainAuthentication,OD,MovedOrRenamed,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/83;https://github.com/jitsi/jicofo/commit/305b5125537f75db0c8f26bae80e42871a943ac6
```
Comment (darko-marinov) [inline pr-data.csv:3947]: `XMPPAuthenticationAuthorityTest.java` was deleted in `305b5125537f75db0c8f26bae80e42871a943ac6`. Please carefully double check `Deleted` vs. `MovedOrRenamed`. You can post somewhere privately justification for each of the four changed lines.

Comment (Carol7102) [reply pr-data.csv]: I have solved merge conflicts and add justifications on CampusWire. Kindly review the changes. Thank you for your attention.

**Pull request merged**